sites/org/pages/docs/[...slug].mjs

```diff
@@ -1,191 +1,7 @@
-// Used in static paths
-import { mdxPaths } from 'site/prebuild/mdx-paths.en.mjs'
-// Dependencies
+import { DocsPage, ns as pageNs } from 'shared/components/docs/docs-page.mjs'
```

Comment:
Same remark. Too abstract. Let pages be pages

Reply:
Yeah, I'm definitely going to need help understanding what the correct level of abstraction is. All docs pages should look and work the same, I feel?